mod-tool/src/main/scala/at/forsyte/apalache/tla/Tool.scala

```diff
@@ -79,11 +79,20 @@ object Tool extends LazyLogging {
     printHeaderAndStatsConfig()
 
     // first, call the arguments parser, which can also handle the standard commands such as version
-    Cli
+    val cli = Cli
       .parse(args)
       .withProgramName("apalache-mc")
       .version("%s build %s".format(Version.version, Version.build), "version")
-      .withCommands(new ParseCmd, new CheckCmd, new TypeCheckCmd, new TestCmd, new ConfigCmd) match {
+      .withCommands(
+          new ParseCmd,
+          new CheckCmd,
+          new TypeCheckCmd,
+          new TestCmd,
+          new ConfigCmd,
+          new ServerCmd
+      )
+
+    cli match {
       // A standard option, e.g., --version or --help. No header, no timer.
```

Comment:
I'm suggesting this temp variable just because the line of the `withCommand` got long enough it wanted wrapping, and I felt it read clearer to break up the CLI parsing clearly from the case analysis that decides how to dispatch based on the result.